shard.yml

```diff
@@ -30,6 +30,6 @@ targets:
   mint:
     main: src/mint.cr
 
-crystal: ~> 1.3.0
+crystal: ~> 1.4.1
```

Comment:
```suggestion
crystal: ~> 1.3
```

Reply:
If you don't want to update to crystal 1.4 maybe just close the PR? I don't get why you'd request this change.

Reply:
There's no need to bump minimal supported Crystal version. It has nothing to do with the _upgrading_ crystal, as it states what versions are supported, and mint support 1.3+.